gnomad_qc/v4/subset.py

```diff
@@ -168,6 +166,19 @@ def main(args):
             metadata=HEADER_DICT,
```

Comment:
Not part of this PR but since were editting it, wondering if it makes sense to add the TODO num-vcf-shards with `parallel` option here to shard the VCF? If needs more thought -- totally get skipping it but seems like a quick todo? Although we'd probably want a tabix option too so maybe this is just another feature request/PR.

Reply:
Yeah, let's make this another PR, this subset script will need more thought anyway for subsets that will have the final annotations also added on.